### PR TITLE
Style Death in Space theme.

### PR DESCRIPTION
**Pull request by mcglincy**

### CODE REVIEW

File: app/assets/stylesheets/themes/deathinspace.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+:root {
+  --dis-black: 0deg 0% 0%;
+  --dis-red: 0deg 100% 30%;
+  --dis-white: 0deg 0% 100%;
+  
+  /* colors.accent */
+  --color--accent-1: var(--dis-green);
```
Comment (jdbann) [inline app/assets/stylesheets/themes/deathinspace.css:7]: Looks like this variable is missing and it makes the highlighted words in the search results appear as black text on a black background

Screenshot 2022-10-29 at 17 26 32

**Review by jdbann — CHANGES_REQUESTED** [review]
Could you also add `death_in_space` to this list of systems: https://github.com/exlibrisrpg/exlibris/blob/main/config/initializers/fixture_helpers.rb#L17

It generates a load of generic seeds which can make it easier to review the design.

Looks great though! And I'm sorry for being so slow to get back to you on this

File: app/assets/stylesheets/themes/deathinspace.css (hunk 1 above)
Comment (mcglincy) [reply app/assets/stylesheets/themes/deathinspace.css:7]: Oops, thanks for the catch!

**Comment by mcglincy** [comment]
I added the system to the fixtures list and re-ran setup. Having all the fixture data visible has exposed some other color issues (e.g., highlighting), so I'm now reviewing all the color vars from the morkborg css and seeing if I need any other Deaith in Space equivalents.

File: app/assets/stylesheets/themes/deathinspace.css (hunk 1 above)
Comment (mcglincy) [reply app/assets/stylesheets/themes/deathinspace.css]: Fixed.

**Comment by mcglincy** [comment]
Added deathinspace to the fixtures systems list and tweaked deathinspace.css to fix black-on-black search terms.

**Review by jdbann — APPROVED** [review]

**Pull request merged**